### PR TITLE
Added troubleshooting section.

### PR DESCRIPTION
**Pull request by mm-s**
Added known problem and workaround (user's permissions for docker daemon).

### CODE REVIEW

**Review by segfaultxavi — CHANGES_REQUESTED** [review]
First off, thanks a lot for submitting a docs pull request!

In the Conversation tab of this PR, at the bottom you'll see a `netlify/nemtech/deploy-preview` line. If you click on the `Details` link you'll see a preview of your PR (after navigating to the page you modified). There you'll spot some rendering errors (like the bullet point in a single line). This is extremely helpful, in case you don't have a local Sphinx installation to check everything before submitting.

The new **Troubleshooting** section is after the **Next steps** section, which should always be the last one.

But foremost, I don't think a single entry warrants a whole new section. In this case, I would resolve this issue with a note at the end of the **Creating a configuration file** section. Something similar to:

```
.. note:: If you get the error ``Permission denied while trying to connect to the Docker daemon socket`` it means that your user does not belong to the docker group. Add it with:

  .. code-block:: bash

    sudo addgroup <YOUR-USERNAME> docker
```

If we start adding too many notes like this one then we can add a **Troubleshooting** section.

**Comment by mm-s** [comment]
Thanks for the review Xavi. I think you're right, a note is more appropriate than creating a new section. I have added your suggested modifications.

File: source/guides/network/using-symbol-bootstrap.rst [hunk 1]
```diff
@@ -110,6 +110,13 @@ Some examples:
 
 This will create a folder called ``target`` by default (It can be changed with the ``-t`` option). Inside there's a folder called ``config`` containing, among other things, the generated complete configuration file (``target/config/preset.yml``) ready to be used to build the network.
 
+.. note:: If you get the error ``Permission denied while trying to connect to the Docker daemon socket`` it means that your user does not belong to the ``docker`` group. Add it with:
+
+  .. code-block:: bash
+  
+    sudo addgroup $USER docker
+
```
Comment (segfaultxavi) [inline source/guides/network/using-symbol-bootstrap.rst:118]:
```suggestion
```

**Review by segfaultxavi — APPROVED** [review]
Awesome, thanks!

**Pull request merged**